.travis.yml

```diff
@@ -1,30 +1,77 @@
 sudo: false
 language: python
-python:
```

Comment:
because that matrix controls the travis systems python

the changes I have submitted set travis to use tox thus we don't need to care about different python versions at the system level as those will be controlled by tox inside it's virtual environments.

I am aware this makes travis config highly coupled to the tox config. I wanted to enforce the maintenance of both configs together.